server/src/main/scala/net/wiringbits/actions/ConfigAction.scala

```diff
@@ -0,0 +1,19 @@
+package net.wiringbits.actions
+
+import net.wiringbits.api.models.Config
+import net.wiringbits.config.ReCaptchaConfig
+
+import java.time.Clock
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class ConfigAction @Inject() (
+    reCaptchaConfig: ReCaptchaConfig
+)(implicit
+    ec: ExecutionContext,
+    clock: Clock
+) {
+  def apply(): Future[Config.Response] = Future {
```

Comment:
Given that the value is already computed, `Future.successful { ... }` must be called instead.